core/src/main/java/com/scalar/db/api/GetBuilder.java

```diff
@@ -1078,7 +1068,7 @@ public BuildableGetFromExistingWithOngoingWhereOr or(AndConditionSet andConditio
   }
 
   public static class BuildableGetFromExistingWithOngoingWhereOr
-      extends BuildableGetFromExistingWithOngoingWhere
+      extends BuildableGetFromExistingWithWhere
```

Comment:
Ah, `BuildableGetFromExistingWithOngoingWhere` implements both  `And<BuildableGetFromExistingWithOngoingWhereAnd>` and `Or<BuildableGetFromExistingWithOngoingWhereOr>`...

I hope Java code analysis or something detects duplicated implementation (in this case, both `BuildableGetFromExistingWithOngoingWhereOr` and `BuildableGetFromExistingWithOngoingWhere` implement `Or<BuildableGetFromExistingWithOngoingWhereOr>`) to notice something.